core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1349,32 +1405,34 @@ public Void visitExchange(ExchangeNode node, Void context)
 
                 addNode(node,
                         format("%sMerge", UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, node.getScope().toString())),
-                        format("[%s]", Joiner.on(", ").join(orderBy)));
+                        ImmutableMap.of("orderBy", orderBy.toString()));
             }
             else if (node.getScope() == Scope.LOCAL) {
                 addNode(node,
                         "LocalExchange",
-                        format("[%s%s]%s (%s)",
-                                node.getPartitioningScheme().getPartitioning().getHandle(),
-                                node.getPartitioningScheme().isReplicateNullsAndAny() ? " - REPLICATE NULLS AND ANY" : "",
-                                formatHash(node.getPartitioningScheme().getHashColumn()),
-                                Joiner.on(", ").join(node.getPartitioningScheme().getPartitioning().getArguments())));
+                        ImmutableMap.of(
+                                "partitionHandle", node.getPartitioningScheme().getPartitioning().getHandle().toString(),
+                                "isReplicateNullsAndAny", String.valueOf(node.getPartitioningScheme().isReplicateNullsAndAny()),
```

Comment:
`String.valueOf(node.getPartitioningScheme().isReplicateNullsAndAny()) -> node.getPartitioningScheme().isReplicateNullsAndAny() ? "true" : ""`
Please use similar pattern wherever we were previously skipping printing a boolean value when it is false

Reply:
This can be a utility function called `formatBoolean`